src/screens/HomePage/utils.ts

```diff
@@ -1,10 +1,26 @@
 import sortBy from 'lodash/sortBy';
 import reverse from 'lodash/reverse';
 import take from 'lodash/take';
-import regions from 'common/regions';
+import { FipsCode, RegionDB } from 'common/regions';
+
+import largestMetroFipsForExploreJson from 'common/data/largest_metro_fips_for_explore.json';
+
+const largestMetroFipsForExplore = largestMetroFipsForExploreJson as FipsCode[];
+
+// This is used to compute data at build time, because it governs a fixed list
+// we can precompute.  So if you need to change this value,
+// re-run `yarn prepare-regions-data` and check in the updated files.
+export const DEFAULT_EXPLORE_GEOLOCATION_COUNT = 5;
 
 /* Gets fips of 5 largest metro areas to set as homepage explore presets */
 export function getLargestMetroFipsForExplore(numRegions: number) {
+  return take(largestMetroFipsForExplore, numRegions);
```

Comment:
Ehh...  I don't think it's worth all of this machinery to generate a list of 5 FIPS that never changes.  It creates some indirection that's going to be hard for somebody to unravel when they come across this code, so I would just remove the `numRegions` parameter (the function comment already implies that this will always be 5) and return `["35620","31080","16980","19100","26420"]` here.  And then undo the rest of this PR.  It's easy enough to write a one-off dummy script to generate different FIPS codes if we want to rework this in the future.  Sorry!

Reply:
no ha ha, this makes sense, thanks for the suggestion : )